### ISSUE TITLE
Friendinator-Android-App-Issue-4

### ISSUE THREAD

**Issue by anirudhkannanvp** [CodeiatioOrganization/Friendinator-Android-App#13]
Create a basic Onboard Screen( 3 Onboard screens ). When Slided right on one onboard screen it should go to the next onboard screen. Name the JS file as issue4.js.

Fork the repository , Add the JS file inside ISSUES-FOLDER.

Commit with message "Friendinator-Issue-1-4" and send a PR.

**Comment by itaniyagupta** [comment]
Assign this issue to me.

**Comment by anirudhkannanvp** [comment]
@thetaniyagupta Done

**Comment by thetaniyagupta** [comment]
> @thetaniyagupta Done

I'll be taking 3 or 4 days more. Is that fine?

**Comment by anirudhkannanvp** [comment]
@thetaniyagupta any updates?

**Comment by itaniyagupta** [comment]
> @thetaniyagupta any updates?

Yeah! I am trying to implement it. Need some time. Is it possible?
